### ISSUE TITLE
Crates.io ownership 📦

### ISSUE THREAD

**Issue by gutenfries** [dart-sys/dart-sys#8]
# Crates.io ownership 📦 

@OptimisticPeach again, apologies for the verbose tagging but we do have a little bit of legwork to do before the repository is ready to be maintained and updated like normal.

In #6, [#6 (comment)](https://github.com/dart-sys/dart-sys/issues/6#issuecomment-1355894694), you had mentioned that you could transfer ownership on [crates.io](https://crates.io).
In order to set up CD, we will need to have publish access to `dart-sys` on [crates.io](https://crates.io).
I am unfamiliar with what [crates.io](https://crates.io) offers as far as team and organization management, but we will need to work something out.

BTW, The university I attend has [crates.io](https://crates.io) blocked on their WiFi network :( I only mention it because this is where I quite often work from (and am as of authoring this issue), so it is not for a lack of competency that I am not versed with the [crates.io](https://crates.io) org management, etc.

Thanks! :)

**Comment by OptimisticPeach** [comment]
Hmm, I'm not actually sure how organization management works on crates.io either -- after some searching it seems that other teams just have multiple owners set to the one crate. Do you happen to have a crates.io account (and if so, link it please, so that I may add you as an owner).

**Comment by gutenfries** [comment]
@/OptimisticPeach I will send you my [crates.io](https://crates.io) details when i get a chance to set it up appropriately.

**Comment by OptimisticPeach** [comment]
Sounds good to me!

**Comment by gutenfries** [comment]
Marking this as `low priority` so we can work on more important things. For the time being, I work mostly remote from uni campus, using a restrictive internet connection ([crates.io](https://crates.io) is blocked ☹️).

also, we are not going to consider CD until the repository is back to production quality.

**Comment by gutenfries** [comment]
@OptimisticPeach My creates.io handle is the same as my GH

```
@gutenfries
```

ty!

**Comment by gutenfries** [comment]
> Hmm, I'm not actually sure how organization management works on crates.io either -- after some searching it seems that other teams just have multiple owners set to the one crate. Do you happen to have a crates.io account (and if so, link it please, so that I may add you as an owner).

@OptimisticPeach, The way Crates.io manages administration rights and code ownership are different.

Only **_ONE_** person (with a valid crates.io account) may possess administrative rights over a crate.
(rights to push to the registry, make releases, etc.)

However, as far as code ownership and acknowledgment go, that is all managed entirely through the `Cargo.toml` in the crate.

If you are comfortable, I propose that we transfer administrative rights to me so that we can publish the work that has been put into Dart-sys, and _of course_, you will still remain a code owner and contributor.

**Comment by OptimisticPeach** [comment]
Hi there!

I just invited you to have ownership of the repository (that was also very easy! they should definitely add some 2fa/validation onto that like how GH has it when doing things of this nature).

I'm not entirely sure how it'll pop up on the crate, but if you try to push and it doesn't work, feel free to remove me as an owner once you've accepted your invite.

Thanks for taking on this project!

**Comment by gutenfries** [comment]
It came through email btw, and now I have publish rights :)

**Comment by OptimisticPeach** [comment]
Souds good!
